<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Calibrate P2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonP">
         <property name="text">
          <string>+ 0.1mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonPP">
         <property name="text">
          <string>+ 1mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonNN">
         <property name="text">
          <string>- 1mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonN">
         <property name="text">
          <string>- 0.1mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelValue">
         <property name="text">
          <string>Current:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonTest">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonExit">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelImage">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
